lib/pure/asyncftpclient.nim

```diff
@@ -146,7 +146,11 @@ proc send*(ftp: AsyncFtpClient, m: string): Future[TaintedString] {.async.} =
   ## Send a message to the server, and wait for a primary reply.
   ## ``\c\L`` is added for you.
   ##
+  ## You need to make sure that the message ``m`` doesn't contain any newline
+  ## characters. Failing to do so will raise ``AssertionDefect``.
```

Comment:
In non-release mode, right? Aren't assertions turned off for release mode?

Reply:
@Varriount doAssert assertions are always enabled